From 2e5ba4538edc08424bea12e0fc0bad54578d8b3e Mon Sep 17 00:00:00 2001
From: Victor Sergeyev <vsergeyev@mirantis.com>
Date: Wed, 10 Jun 2015 14:44:59 +0300
Subject: [PATCH] Get mox from mox3, not from six.moves

oslotest 1.7.0 breakes oslo.messaging unittests, because it doesn't add
mox to six.moves anymore - see change Ic59c73abb9b09cb594bf7df4173d7f99f81d526c

This patch fixes imports on test run.

Change-Id: Ia40b7733bd2e74b1a1703f2e1a5245f01debbd36
---
 oslo_messaging/tests/test_transport.py | 2 +-
 tests/test_transport.py                | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/oslo_messaging/tests/test_transport.py b/oslo_messaging/tests/test_transport.py
index a09953a..8fad947 100644
--- a/oslo_messaging/tests/test_transport.py
+++ b/oslo_messaging/tests/test_transport.py
@@ -14,9 +14,9 @@
 #    under the License.
 
 import fixtures
+from mox3 import mox
 from oslo_config import cfg
 import six
-from six.moves import mox
 from stevedore import driver
 import testscenarios
 
diff --git a/tests/test_transport.py b/tests/test_transport.py
index 5a9a73c..36f8db8 100644
--- a/tests/test_transport.py
+++ b/tests/test_transport.py
@@ -14,8 +14,8 @@
 #    under the License.
 
 import fixtures
+from mox3 import mox
 import six
-from six.moves import mox
 from stevedore import driver
 import testscenarios
 
